gradle.properties

```diff
@@ -1,9 +1,17 @@
-ideaVersion=IC-2018.3
+ideaVersion=IC-2019.3
```

Comment:
Is that your lowest version now ? In that case you need to update IJ.yml

Reply:
removed IC-2018.3 in IJ.yml